hardware_interface/include/hardware_interface/handle.hpp

```diff
@@ -77,16 +60,11 @@ class ReadOnlyHandle
 class ReadWriteHandle : public ReadOnlyHandle
 {
 public:
-  ReadWriteHandle(
-    const std::string & name, const std::string & interface_name, double * value_ptr = nullptr)
+  ReadWriteHandle(const std::string & name, const std::string & interface_name, double * value_ptr)
```

Comment:
Got to throw on nullptr in this constructor too

Reply:
Not really because we call another constructor from it. Isn't it?

Reply:
Correct! I take it back